src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+import duke.tasks.*;
+import duke.exception.DukeException;
+
+/**
+ * Provides functionality for reading and writing from storage file.
+ */
+public class Storage {
+    private File data;
+
+    public Storage(String s) {
+        try {
+            this.data = new File(s);
+        } catch (Exception e) {
+            System.err.println("Error retrieving file!");
+        }
+    }
+
+    /**
+     * Builds a TaskList class from storage Filepath.
+     * @return Returns a TaskList class built according to storage Filepath. Returns an empty TaskList class
+     * if there is no storage file.
+     * @throws DukeException Filepath not found.
+     */
```

Comment:
Based on the coding standards [here](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html), there should be an empty line between description and parameter section.